content/articles/2017-11-21-usethis-1-0-0.Rmd

```diff
@@ -0,0 +1,144 @@
+
+(If you find yourself using usethis a lot and don't want to load it everytime, run `use_usethis()` and follow the instructions.)
+
+Notice that the output includes "Changing active project to mypkg". usethis has the concept of an "active project" which is where all usethis functions will have effect. It is usually the working directory, but is changed automatically by the `create_`  functions, and can be changed manually with `proj_set()`.
+
+If you don't want a package, use `create_project()` to create a minimal [RStudio project](https://support.rstudio.com/hc/en-us/articles/200526207-Using-Projects). usethis is gradually evolving towards supporting more general R "projects" but this is still a work in progress. Please [let us know](https://github.com/r-lib/usethis/issues) if you have ideas or discover a problem!
+
+### Code and tests
+
+`use_r()` will create/edit a script file in `R/`:
+
+```{r}
+use_r("foo")
+```
+
+`use_test("foo")` will set up all the infrastructure you need for [unit testing](http://r-pkgs.had.co.nz/tests.html), and create/edit a test file in `tests/teststhat/`:
+
+```{r}
+use_test("foo")
+```
+
+We recommend that you have paired test and source files. If you use RStudio, you can use `use_test()` to create/edit the test file corresponding to the open source file, and `use_r()` to create/edit to the source file corresponding to the open test file. 
+
+### Dependencies
+
+To declare that your project has some dependencies, use `use_package()`:
+
+```{r}
+use_package("ggplot2")
+use_package("dplyr")
+```
+
+Or if you want a dependency on a development version of a package, use `use_dev_package()`.
+
+### Documentation
+
+Take advantage of these helpers to document your package:
+
+* `use_roxygen_md()` sets up [roxygen2](http://r-pkgs.had.co.nz/man.html) and
+  enables markdown mode so you can use markdown in your roxygen2 comment blocks.
+
+* `use_package_doc()` creates a skeleton documentation file for the complete
+  package, taking the advantage of the latest roxygen2 features to minimise
+  duplication between the `DESCRIPTION` and the documentation.
+  
+* `use_readme_rmd()` creates a
+  [`README.Rmd`](http://r-pkgs.had.co.nz/release.html#important-files): 
+  use this to describe what your package does and why people should care about 
+  it.
+  
+* `use_news_md()` creates a basic `NEWS.md` for you to record changes.
+
+* `use_vignette("vignette-name")` sets you up for success by configuring
+  `DESCRIPTION` and creating a `.Rmd` template in `vignettes/`
+
+### Sharing 
+
+If you want to share your code with others, it's good practice to make the licensing clear. usethis provides four helpers for four most common open source licenses:
```

Comment:
"for THE four ..."

Reply:
Could do: "usethis provides helpers for the four most common...", thereby eliminating the need for two fours?